server/bin/pbench-dispatch.sh

```diff
@@ -3,222 +3,284 @@
 
 # This script is the first part of the pipeline that processes pbench
```

Comment:
Reviewing `pbench-dispatch.sh` is really not a simple set of diffs, but making sure `pbench-server-prep-shim.sh` behaviors landed properly into `pbench-dispatch.sh`.  It is more of a "lift-n-shift" type review than a pure "change" review.